app/services/platform-apps/index.ts

```diff
@@ -446,7 +447,8 @@ export class PlatformAppsService extends
     return fetch(request)
       .then(handleErrors)
       .then(res => res.json())
-      .then(json => json.id_hash);
+      .then(json => json.id_hash)
+      .catch(() => null);
```

Comment:
Do we need to handle this one in a special place somewhere else?  We should avoid loading an app with an app id of `null`.  That will cause a lot of issues for us down the line.  The app id is pretty important.